common/orderData.js

```diff
@@ -0,0 +1,15 @@
+class OrderData {
+    getData() {
+        return fetch("/data/orderUp.json")
+        .then(stream => stream.body.getReader().read())
+        .then(({value, done}) => this.parse(value));
+    }
+
+    parse(value) {
+        const string = value.reduce((acc, code) => acc += String.fromCharCode(code), "");
+        return JSON.parse(string).map(e => "\n"+e+"\n");
+    }
+
+}
+
+window.OrderData = OrderData;
```

Comment:
the getData can definitely go into ExampleOrder, but what about making the parse function into a Stream Transformer?